features/networking/ptp_operator.feature

```diff
@@ -11,3 +11,101 @@ Feature: PTP related scenarios
       | app=linuxptp-daemon |
     And status becomes :running of exactly 1 pods labeled:
       | name=ptp-operator |
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-25738
+  @admin
+  Scenario: Linuxptp run as slave can sync to master
+    Given the ptp operator is running well
+    Given I store the nodes in the clipboard
+    Given I obtain test data file "networking/ptp/ptp_master.yaml"
+    When I run oc create as admin over "ptp_master.yaml" replacing paths:
+      | ["spec"]["recommend"][0]["match"][0]["nodeName"]      | <%= cb.nodes[0].name %> |
```

Comment:
Extra white spaces between `["nodeName"]` and `|`.
Same issue in below tables.

Reply:
fixed it, thanks!